File: datasets/cs-keep-code_davinci_001_temp_0.2/HumanEval_122_add_elements.cs
using System;
using System.Numerics;
using System.Diagnostics;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Security.Cryptography;
class Problem {
    // Given a non-empty array of integers arr and an integer k, return
    // the sum of the elements with at most two digits from the first k elements of arr.
    // Example:
    // Input: arr = [111,21,3,4000,5,6,7,8,9], k = 4
    // Output: 24 # sum of 21 + 3
    // Constraints:
    // 1. 1 <= len(arr) <= 100
    // 2. 1 <= k <= len(arr)
    public static int AddElements(List<int> arr, int k) {
        int sum = 0;
        for (int i = 0; i < k; i++) {
            if (arr[i] < 100) {
                sum += arr[i];
            }
        }
        return sum;

    }
    public static void Main(string[] args) {
    Debug.Assert(AddElements((new List<int>(new int[]{(int)1, (int)-2, (int)-3, (int)41, (int)57, (int)76, (int)87, (int)88, (int)99})), (3)) == (-4));
    Debug.Assert(AddElements((new List<int>(new int[]{(int)111, (int)121, (int)3, (int)4000, (int)5, (int)6})), (2)) == (0));
    Debug.Assert(AddElements((new List<int>(new int[]{(int)11, (int)21, (int)3, (int)90, (int)5, (int)6, (int)7, (int)8, (int)9})), (4)) == (125));
    Debug.Assert(AddElements((new List<int>(new int[]{(int)111, (int)21, (int)3, (int)4000, (int)5, (int)6, (int)7, (int)8, (int)9})), (4)) == (24));
    Debug.Assert(AddElements((new List<int>(new int[]{(int)1})), (1)) == (1));
    }

}

File: datasets/scala-keep-code_davinci_001_temp_0.2/HumanEval_122_add_elements.scala
import scala.math._
import scala.collection.mutable._
object Problem {
    // Given a non-empty array of integers arr and an integer k, return
    // the sum of the elements with at most two digits from the first k elements of arr.
    // Example:
    // Input: arr = [111,21,3,4000,5,6,7,8,9], k = 4
    // Output: 24 # sum of 21 + 3
    // Constraints:
    // 1. 1 <= len(arr) <= 100
    // 2. 1 <= k <= len(arr)
    def addElements(arr : List[Int], k : Int) : Int = {
        var sum = 0
        for (i <- 0 until k) {
            sum += arr(i)
        }
        return sum

    }
    def main(args: Array[String]) = {
    assert(addElements((List[Int](1.toInt, -2.toInt, -3.toInt, 41.toInt, 57.toInt, 76.toInt, 87.toInt, 88.toInt, 99.toInt)), (3)) == (-4));
    assert(addElements((List[Int](111.toInt, 121.toInt, 3.toInt, 4000.toInt, 5.toInt, 6.toInt)), (2)) == (0));
    assert(addElements((List[Int](11.toInt, 21.toInt, 3.toInt, 90.toInt, 5.toInt, 6.toInt, 7.toInt, 8.toInt, 9.toInt)), (4)) == (125));
    assert(addElements((List[Int](111.toInt, 21.toInt, 3.toInt, 4000.toInt, 5.toInt, 6.toInt, 7.toInt, 8.toInt, 9.toInt)), (4)) == (24));
    assert(addElements((List[Int](1.toInt)), (1)) == (1));
    }

}
